obelisk_ws/src/cpp/obelisk_control_cpp/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
do we need a license file here? i think we already have one at the top level

Reply:
this came default in the ros package. I just assumed that this was standard practice to have one in every ROS package, but we can remove it if we want.